script.js

```diff
@@ -1,11 +1,116 @@
 /* eslint-disable linebreak-style */
 const hamburger = document.querySelector('.hamburger');
 const navMenu = document.querySelector('.desktop-navigation');
+const body = document.querySelector('body');
+const header = document.querySelector('header');
+
 hamburger.addEventListener('click', () => {
   hamburger.classList.toggle('active');
   navMenu.classList.toggle('active');
+  header.classList.toggle('full-screen');
+  body.classList.toggle('no-scroll');
 });
 document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
   hamburger.classList.remove('active');
   navMenu.classList.remove('active');
-}));
\ No newline at end of file
+  header.classList.remove('full-screen');
+  body.classList.remove('no-scroll');
+}));
+
+// const arr = ['hello', 'hi', 'bye'];
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly remove this line.

Reply:
Hi @leonardodiasb  
I am trying to push my changes to this project but it is saying conflicts with the main branch.
please help how to resolve this.